techmatters/hrm
# CHI-1248 use swagger-json for inline swagger definitions

## Description
This updates swagger-ui-express to eliminate security issues, adds swagger-json, and splits the routes currently covered by swagger into inline comment based schemas/definitions.  

Even splitting the definitions up into comments, swagger definitions are not "lightweight". 

I'm not sure how far I am expected to go with updating all routes. I can continue if this is a priority, but it is fairly time consuming. I'm also going to do another round of double checking that what I have documented is correct. But this gives a general idea of what it looks like. 

I'm going to leave this as a WIP until I get buy in that this is the right direction and clarification around how many endpoints we'd like to cover in this first iteration.

### Related Issues
CHI-1248

### Verification steps
Run your local hrm dev env and check http://localhost:8080/swagger to verify documentation.
